### ISSUE TITLE
How to search by custom id field with select.row?

### ISSUE THREAD

**Issue by CaptainChemist** [reactabular/reactabular#213]
Thanks for the great project! I can't quite figure out how to use the select.row when the id is custom. My `id` is `_id`, how do I modify this setState function to look among the `_id`s instead of `id`? I know it says that you can do it in the manual, but I just couldn't figure out how from the code. 

This is the sample code in the documentation. My initial guess was to sub in `rows[selectedRowIndex]._id` but that doesn't seem to work.
```
    this.setState(
      select.row({
        rows,
        selectedRowId: rows[selectedRowIndex].id
      })
    );
```

I can "fix" the issue by going into the row.js file and changing to `row._id === selectedRowId` on line 13 but that seems like a bad idea.

**Comment by bebraw** [comment]
Hi,

I think you need to override `isSelected`. See [the full source](https://github.com/reactabular/reactabular/blob/master/packages/reactabular-select/src/row.js) for a better idea of what's going on there.

**Comment by CaptainChemist** [comment]
Yes, I have looked at that particular file and that is where I saw on `line 3` that it looks for `row.id === selectedRowId` instead of say `row[customKeyName] === selectedRowId` with the ability of passing in `customKeyName` to define the custom key name. I am really sorry but I think I am failing to understand how I can use `isSelected` to pass in a custom key name. How would I change the `select.row` call in my first post to pass in the custom key name without resorting to changing the `row.js` file?  Thanks so much!
